2-WebApp-graph-user/2-2-TokenCache/Startup.cs

```diff
@@ -52,10 +45,10 @@ public void ConfigureServices(IServiceCollection services)
              */
 
             services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
-                    .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
-                        .EnableTokenAcquisitionToCallDownstreamApi(initialScopes)
-                            .AddMicrosoftGraph(Configuration.GetSection("DownstreamApi"))
-                            .AddDistributedTokenCaches();
+                .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))

```

Comment:
Actually the indentation is really based on the builders ...

Reply:
reverted